index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <h3>The man who saved a billion lives</h3>
+        </div>
+        <div class="img-div">
+            <img id="img-caption" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="photography">
+            <h3>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part
+                of his life-long war on hunger.</h3>
+        </div>
```

Comment:
It would be better if you use a header tag here instead of a div as there should be more use of semantic tags for best HTML practices.